app/models/sorn.rb

```diff
@@ -112,6 +116,17 @@ def section_snippets(selected_fields, search_term)
     output
   end
 
+  def update_mentioned_sorns
+    mentioned_sorns_in_xml.each do |child_sorn|
```

Comment:
you'd stated in a meeting you weren't sure about the term mentioned. FWIW, I do think it works, although if we wanted to we could consider "referenced" as an alternative verb.

Reply:
I think I'll leave it at mentioned, because `references` is an existing method for Rails. It's not an official reserved word, but it's close enough that I would get confused.

Reply:
👍